src/main/java/graphql/execution/FieldCollector.java

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
Update file to bring in line with v21 - it's because ConditionalNodes are referenced in the ExecutableNormalizedOperationFactory, and those changes are viral in a few other files